test/test_toolbar.py

```diff
@@ -7,7 +7,6 @@
 
 def load_app(name):
     app = __import__(name).app
-    app.config['TESTING'] = True
```

Comment:
I'm not sure this should be removed, but best to discuss/research in a separate PR that is specific to this `TESTING`/setting `DEBUG`...